### PR TITLE
Release 1.11

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: docker/build.sh [hunk 1]
```diff
@@ -18,16 +18,20 @@ esac
...
+FIRST_TAG=${1:-latest}
+LOCAL_TAG=openmotics/frontend:$FIRST_TAG
 mv ../dist.tgz .
+docker build -t $LOCAL_TAG .
 
-docker build -t $TAG .
-docker tag $TAG $AWS_TAG
 if [ "${PUSH}" ] ; then
-	docker push $AWS_TAG
+        $(aws ecr get-login --no-include-email)
```
Comment (svanscho) [inline docker/build.sh]: 👍

File: src/components/shared.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ class Shared {
             this.locale = undefined;
             this.installation = undefined;
             this.autoLogin = undefined;
+            this.current_user = null;
```
Comment (svanscho) [inline src/components/shared.js]: @khenderick what's the reason for using null vs undefined on current_user?

Comment (khenderick) [reply src/components/shared.js]: Crap, it's indeed something I missed in a previous PR, will correct that on dev in a minute. Nice catch!

Comment (khenderick) [reply src/components/shared.js]: Turns out the variable isn't used anymore. I'll take it up as a separate change. It's not nice but doesn't break anything.

**Pull request merged**